available-connectors.json

```diff
@@ -191,6 +201,42 @@
     "supportedByStrongLoop": true
   },
   {
+    "name": "kv-redis",
+    "description": "Redis key-value connector",
+    "baseModel": "KeyValueModel",
+    "features": {
+      "discovery": false,
+      "migration": false
+    },
+    "settings": {
+      "url": {
+        "type": "string",
+        "description": "Connection String url to override other settings (eg: redis://user:pass@host:port/db)"
+      },
+      "host": {
+        "type": "string"
+      },
+      "port": {
+        "type": "number"
+      },
+      "user": {
+        "type": "string"
+      },
+      "password": {
+        "type": "string",
+        "display": "password"
```

Comment:
Since the setting name is already `password`, there is no need for `display` to say the same again.